firefoxThemes/options/options.html

```diff
@@ -7,19 +7,71 @@
   <link rel="stylesheet" href="option.css"/>
 </head>
 <body>
+<h1>Settings</h1>
 <div class="options-container">
-  <section>
-    <span id="loadOnStart" role="checkbox"></span>
-    <label for="loadOnStart">Load Doki Theme page on startup</label>
+  <!--General Settings-->
+  <section id="general">
+    <h2>General</h2>
+    <section>
+      <span id="loadOnStart" role="checkbox"></span>
+      <label for="loadOnStart">Load Doki Theme page on startup</label>
+    </section>
+    <section>
+      <span id="textSelection" class="checked" role="checkbox"></span>
+      <label for="textSelection">Text Selection</label>
+    </section>
+    <section>
+      <span id="scrollbar" class="checked" role="checkbox"></span>
+      <label for="scrollbar">Scrollbar</label>
+    </section>
   </section>
-  <section>
-    <span id="textSelection" class="checked" role="checkbox"></span>
-    <label for="textSelection">Text Selection</label>
-  </section>
-  <section>
-    <span id="scrollbar" class="checked" role="checkbox"></span>
-    <label for="scrollbar">Scrollbar</label>
+  <!--System Settings-->
+  <section id="theme">
+    <h2>System Theme</h2>
+    <div>
+      <span id="system" role="radio"></span>
+      <label for="system">System</label>
```

Comment:
System theme is a bit confusing to me, 

What about

## Theme Settings
- Match System
- Druthers
- Dark Only
- Light Only
- All Themes

What do you think?

Reply:
I agree with the title change to *Theme Settings* alongside the other option names, however I'm not too sure about the name of *Match System*. 

What about *Match Device*? This sound more clear. *To me at least.*

Reply:
Here's what the System Theme settings look like now:
## Theme Settings
- Match Device 
- Druthers
- Dark Only
- Light Only
- All

@Unthrottled What do you think? A tooltip is also included, which will provide a brief description of their functionality.

Reply:
Device works for me. Though when I see `Device` I think of mobile phones, and my physical computer. I see System more as the software that is running on my devices. That being said, I think the meaning is still conveyed, with either name.